app/views/subscriptions/_subscription.html.haml

```diff
@@ -14,7 +14,7 @@
   - unless subscription.tag_list.empty?
     %p
       = t('.only_persons_with')
-      - subscription.tag_list.each do |tag|
-        %span.label.label-inverse.person-tag= tag
+      - subscription.tags.each do |tag|
```

Comment:
I don't understand why this change from `tag_list` to `tags` is necessary. Does `tag_list` only contain the names while `tags` returns the `Tag` models? Or what else is the reason?

Reply:
yes, that's exactly the difference! we need instances of Tag for the translation helper